app/views/components/blockEditor.html

```diff
@@ -114,34 +113,39 @@
               </label>
             </div>
           </div>
-          <p ng-if="!canChangeRegTypes" class="text-muted"><i class="fa fa-exclamation-triangle"></i> <translate>This question must be shown for all registrant types.</translate></p>
+          <p ng-if="!canChangeRegTypes" class="text-muted"><i class="fa fa-exclamation-triangle"></i>
+            <translate>This question must be shown for all registrant types.</translate>
+          </p>
         </div>
         <div class="form-group" ng-if="hasOptions">
           <label class="inline-label" translate>Expense Type</label>
-             <select class="form-control" ng-model="block.expenseType" ng-options="key as value for (key , value) in expenseTypesConstants">
+          <select class="form-control" ng-model="block.expenseType" ng-options="key as value for (key , value) in expenseTypesConstants">
               <option value="" translate>None</option>
              </select>
         </div>
       </tab>
       <tab heading="Rules" ng-if="canHaveRules">
-        <p class="text-muted"><i class="fa fa-info-circle"></i> <translate>Rules are a great way to hide questions based on the registrant's answers to other questions. All rules created must be met before the question is displayed.</translate>
-          <translate>Rules can depend on questions of</translate> <a href="" tooltip="{{'Multiple Choice, Dropdown, Number, Date, Gender or Year in School' | translate}}" translate>these types</a> <translate>that are above this question.</translate></p>
+        <p class="text-muted"><i class="fa fa-info-circle"></i>
+          <translate>Rules are a great way to hide questions based on the registrant's answers to other questions. All rules created
+            must be met before the question is displayed.</translate>
+          <translate>Rules can depend on questions of</translate> <a href="" tooltip="{{'Multiple Choice, Dropdown, Number, Date, Gender or Year in School' | translate}}"
+            translate>these types</a>
+          <translate>that are above this question.</translate>
+        </p>
 
```

Comment:
@OzzieOrca We'll see to it that we do not format the whole page, but instead format only the code created / modified by us.